optimize/backend/src/main/java/io/camunda/optimize/rest/security/platform/PlatformSecurityConfigurerAdapter.java

```diff
@@ -143,7 +142,6 @@ protected SecurityFilterChain configureWebSecurity(HttpSecurity http) {
                     // common public api resources
                     .requestMatchers(
                         new AntPathRequestMatcher(createApiPath(READYZ_PATH)),
-                        new AntPathRequestMatcher(createApiPath(STATUS_PATH)),
```

Comment:
no problem with changing this class, but also want to make sure that it is on your radar that this class can be deleted completely?

Reply:
yup, anything C7 only and anything that uses CamundaPlatformCondition (incl the condition itself) will be removed ♻️ the adjustment here was just since I deleted the constant